### ISSUE TITLE
"Illegal Instruction" crash on Linux when printing text with CRLF inside

### ISSUE THREAD

**Issue by vzsg** [vapor/leaf#41]
Consider the following code (replacing the "It works" text of the [basic](https://github.com/vapor/basic-template) template):

```swift
let drop = Droplet()

drop.get { req in
    return try drop.view.make("welcome", [
    	"content": "\r\n\""
    ])
}
```

This code works fine on macOS, but causes the process to crash on Linux, both on a physical Ubuntu installation and in Docker. The only output is "Illegal Instruction".

It seems that strings that match the following pattern: `\r\n.*["|']` – in other words, strings that contain a Windows newline and a single or double quote – trigger the crash when printed into the output using the `#()` tag.

**Comment by jameslinjl** [comment]
I ran into this same bug this past weekend and chatted with @LoganWright about it. Submitted a PR with a test that repros the issue: https://github.com/vapor/leaf/pull/40

**Comment by jameslinjl** [comment]
Logan said that he'd try to look at it sometime today

**Comment by vzsg** [comment]
Yep, this is your bug actually :) I would have linked your repro repo too, if it was still accessible. The new test is much better though.

**Comment by LoganWright** [comment]
@vzsg @jameslinjl 

I'm hashing away at this right now, not making much progress so far. Not sure what specifically about `\r\n\"` causes a crash only on linux and not getting much reporting. 

If anyone sees this and recognizes that pattern as somehow significant, please let us know in case there's a hint of how to handle it.

**Comment by vzsg** [comment]
This is where the crash is coming from:

```
Core was generated by `.build/debug/App'.
Program terminated with signal SIGILL, Illegal instruction.
#0  0x00007fc39e32a2df in _TTSf4s_gs_n___TFC10Foundation15NSMutableString17replaceCharactersfT2inVS_8_NSRange4withSS_T_ () from /usr/lib/swift/linux/libFoundation.so
[Current thread is 1 (Thread 0x7fc3939e5700 (LWP 2948))]
```

**Comment by vzsg** [comment]
Okay, I got something from the stacktrace:

```swift
import Foundation
let str = "\r\n\"\"".replacingOccurrences(of: "\"", with: "T")
```

Same crash.

**Comment by vzsg** [comment]
Fixed by #42, so I'm closing the issue.
